src/OrchardCore.Modules/OrchardCore.Media/Processing/MediaImageSharpConfiguration.cs

```diff
@@ -27,17 +27,17 @@ public void Configure(ImageSharpMiddlewareOptions options)
             {
                 // Force some parameters to prevent disk filling.
                 // For more advanced resize parameters the usage of profiles will be necessary.
-                // This can be done with a custom IImageWebProcessor implementation that would 
+                // This can be done with a custom IImageWebProcessor implementation that would
                 // accept profile names.
 
-                validation.Commands.Remove(ResizeWebProcessor.Compand);
-                validation.Commands.Remove(ResizeWebProcessor.Sampler);
-                validation.Commands.Remove(ResizeWebProcessor.Xy);
-                validation.Commands.Remove(ResizeWebProcessor.Anchor);
-                validation.Commands.Remove(BackgroundColorWebProcessor.Color);
-
                 if (validation.Commands.Count > 0)
                 {
+                    validation.Commands.Remove(ResizeWebProcessor.Compand);
```

Comment:
No use to do remove calls when we know the set is empty, seemed to be the case with blog theme at least

Reply:
Cool !